src/main/java/io/wispforest/lavender/structure/StructureTemplate.java

```diff
@@ -61,53 +87,149 @@ public StructureTemplate(Identifier id, BlockStatePredicate[][][] predicates, in
         }
     }
 
-    /**
```

Comment:
Further, why did you seemingly reorder half of this class' methods (which also means they longer match up with the comments that denote the sections)? For one it's, again, unrelated to implementing features *and* it makes the diff basically unreadable

Reply:
my autoformatter *really* wanted to reorder them lol (I've configured my autoformatter to order methods in a consistent way, just bc imo it's codesmell to have static methods mixed in with getters and shit, so I want my autoformatter to enforce that. not saying you did that, just that's why I have them)

also, I did try to move the comments with the methods, so that they would stay consistent.

but, along with the other requested changes, I'm going to rewrite my commits and then force-push to my branch so that they're back in their original place, to restore the diff and not include it in the commit history.